Makefile

```diff
@@ -22,4 +22,10 @@ config/local.sh:
 clean:
 	rm -rf reports/
 
-.PHONY: smoke-tests run rerun setup install clean
+docker-up:
+	$(eval export AWS_ACCESS_KEY_ID=$(shell aws configure get aws_access_key_id))
+	$(eval export AWS_SECRET_ACCESS_KEY=$(shell aws configure get aws_secret_access_key))
+	$(eval export DM_MANDRILL_API_KEY=$(shell ${DM_CREDENTIALS_REPO}/sops-wrapper -d ${DM_CREDENTIALS_REPO}/vars/preview.yaml | grep mandrill_key | sed 's/^.* //'))
```

Comment:
To get the containers up and running it's not strictly necessary to set any of these. You just won't be able to upload documents or send fake emails. So the func tests will fail. I have a feeling that some people won't mind if this fails, they just want to play with the apps. I guess they could just use `docker-compose up` in that case.